src/core_plugins/elasticsearch/lib/create_data_cluster.js

```diff
@@ -0,0 +1,30 @@
+export default function (server) {
+  const config = server.config();
+  const esPlugins = server.plugins.elasticsearch;
+  const Logger = server.plugins.elasticsearch.ElasticsearchClientLogging;
```

Comment:
Like my previous comment, how about destructuring this rather than renaming the variable?

```js
const { ElasticsearchClientLogging } = server.plugins.elasticsearch;

class DataClientLogging extends ElasticsearchClientLogging
```